Dockerfile

```diff
@@ -10,6 +10,12 @@ RUN apt-get update; \
   unison netcat net-tools memcached nano libapache2-mod-php php php-cli php-common \
   php-gd php-json php-mbstring php-xdebug php-mysql php-opcache php-curl \
   php-readline php-xml php-memcached php-oauth php-bcmath; \
+  echo "postfix postfix/mailname string drupal-mail" | debconf-set-selections; \
```

Comment:
could we move these lines up so that `apt-get install --assume-yes postfix mailutils`gets included in the list o packages above?

Reply:
sure, done